File: fun_schedule.js
/*
** Randomly generates a fun schedule.
** Script by: tailsjs
*/

function random(min, max) {return Math.round(Math.random() * (max - min)) + min} // from https://github.com/nitreojs/israelcods/blob/master/random.js

let array = "wake up,go to school,teach js,play cs:go with onetap,become boomer".split(',') // comma separated actions
let string = ``
for(let i = 0; i < random(3,13); i++)string += `${i + 1}. ${array[random(0,array.length - 1)]}\n`
return console.log(string)

File: FunSchedule.cs
namespace FunSchedule
{
    class Program
    {
        static void Main()
        {
            Random rng = new Random();
            string[] ListOfNeed = "wake up,go to school,teach c#,play cs:go with onetap,become boomer".Split(",");
            string result = "";
            for(var i = 0; i < rng.Next(3, 13); i++)
            {
                result += $"{i + 1}. {ListOfNeed[rng.Next(ListOfNeed.Length)]}\n";
            }
            Console.WriteLine(result);
        }
    }
}